src/SonarScanner.MSBuild.Common/CommandLine/CommandLineFlagPrefix.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SonarScanner.MSBuild.Common.CommandLine
+{
+    public static class CommandLineFlagPrefix
+    {
+        private static readonly char[] _prefixes = new[] { '-', '/' };
+
+        public static string[] GetPrefixFlag(string[] flags)
```

Comment:
`GetPrefixedFlags` would be a more accurate name.

Reply:
Agreed, fixed.